os/mm/mm_heap/mm_mallinfo.c

```diff
@@ -73,9 +87,112 @@
  * Private Functions
  ****************************************************************************/
 
+static void dump_node(struct mm_allocnode_s *node, node_type_t type)
+{
+	if (type == TYPE_CORRUPTED) {
```

Comment:
Very minor comment, When you checking type of enum, you'd better use switch-case for expandability.
If there's no chance that expand node_type_e, please ignore this comment.